onnxruntime/core/graph/graph_utils.cc

```diff
@@ -399,6 +399,11 @@ bool IsGraphInput(const Graph& graph, const NodeArg* input) {
   return std::find(graph_inputs.begin(), graph_inputs.end(), input) != graph_inputs.end();
 }
 
+bool IsNodeArgConstant(const Graph& graph, const NodeArg& node_arg) {
+  const onnx::TensorProto* initializer = nullptr;
+  return graph.GetInitializedTensor(node_arg.Name(), initializer) && !IsGraphInput(graph, &node_arg);
+}
```

Comment:
How about rename the function name to IsNodeInputConstant? Also, the function AllNodeInputsAreConstant@graph_utils.cpp can be modified to call this function.

Reply:
I would leave the name as is, given that the NodeArg can be either an input or an output (or both depending whether it's the downstream or the upstrem node).
Sure, I can change AllNodeInputsAreConstrant to use this function.